### ISSUE TITLE
Nitrobenzene issues

### ISSUE THREAD

**Issue by chochem** [GTNewHorizons/GT-New-Horizons-Modpack#10650]
### Your GTNH Discord Username

chochom#0271

### Your Pack Version

2.1.2.4-17-pre

### Your Server

SP

### Type of Server

Single Player

### Your Expectation

-

### The Reality

1. version .4 added an unintended Nitrobenzene recipe in the LCR. This should be removed. The chem plant is the intended method. Its also cheaper in terms of ingredients as it just uses water not distilled water.
(open)
3. dev17 nerfed the chem plant recipe (for chemical standardization) requiring more nitric acid. Nitrobenzene should get an EU buff to compensate. Nitrobenzene is already underused as a fuel.
(fixed by https://github.com/GTNewHorizons/GTplusplus/pull/237)
4. hilariously dev17 did not nerf the LCR recipe, so now the recipes are genuinely different too. But yeah, see 1, just remove it.
(fixed by https://github.com/GTNewHorizons/GT5-Unofficial/pull/1062) 

### Your Proposal

remove LCR recipe for nitrobenzene. buff EU fuel value.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by xSkewer** [comment]
with hss-g coils, nitrobenzene recipe is 12s at hv

assuming 4 parallels, 1666.66 nitrobenzene/s costing 1000 nitric/s

with the nerfed recipe, 1666.66 nitrobenzene/s costs 1666.66 nitric/s

at 1666.66 nitric/s with old recipe, this is over 6 parallels, meaning a loss of ~1000 nitrobenzene/s assuming the others inputs are maintained

at 100% efficiency, 1666.66 nitrobenzene/s is ~104166.25eu/t
if you supplied the amount of nitric for the nerfed recipe but with the pre-nerf recipe (and all the other inputs), you would get an additional ~62500eu/t

one bucket of nitric = 132480eu assuming lcrs
factoring in the cost of producing an additional 666.66 nitric/s, this loses an additional ~4416eu/t.

therefore, a re-buff to nitrobenzene would mean that 1666.66 nitrobenzene/s produces ~166666.25eu/t rather than ~104166.25eu/t. we can bring this up to ~180keu/t to account for the costs of producing hydrogen/nitrogen/oxygen for the extra nitric as well as the production of the extra nitric. 

producing 180keu/t with 1666.66 nitrobenzene/s (83.333 nitrobenzene/t) means nitrobenzene's fuel value should be brought up to 2160 from 1250. while this sounds crazy, this is brings it in line with the old recipe since reducing the nitric input required is not an option.

the math here does ignore the additional cost of the other inputs, but its not like nitrobenzene was breaking anything so an overall buff should be fine.

tldr: Increase Nitrobenzene's fuel value to 2160 from 1250 to compensate for the increased nitric acid requirements and also buff Nitrobenzene overall since it isn't very appealing.

**Comment by miozune** [comment]
Standardization of GT recipes is WIP, hold on

**Comment by miozune** [comment]
Fixed by https://github.com/GTNewHorizons/GT5-Unofficial/pull/1062

**Comment by xSkewer** [comment]
2 and maybe 1 have not been addressed.
